src/wrapper.rs

```diff
@@ -112,9 +113,24 @@
 /// }
 /// ```
 ///
+/// #### Non-derivable classes
+///
+/// By convention, GObject implements "final" classes, i.e. those who cannot
+/// be subclassed, by exposing a public Instance struct, but no corresponding
```

Comment:
Note that the instance struct could also be opaquely defined (i.e. only a pointer typedef in the public header, with the actual struct definition hidden). This is not a problem though and handled